robot_skills/test/lazy_subscriber_test.py

```diff
@@ -0,0 +1,19 @@
+# ROS
```

Comment:
This should be removed from the test folder and move somewhere else. The test folder will be used for CI tests only.